.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
 env:
 - TEST_DIR=.
 - TEST_DIR=scripts/babel-relay-plugin
-script: cd $TEST_DIR && npm install && npm test
+script: cd $TEST_DIR && npm install && npm run lint && npm test
```

Comment:
This isn't going to work since `babel-relay-plugin` doesn't have that script defined. I think we'll probably actually want a better way to do this (maybe a `travis-test` task in each that gets run, looks sort of like relay's `test` to run multiple things)